inference-engine/src/transformations/src/itt.hpp

```diff
@@ -0,0 +1,66 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+/**
+ * @brief Defines openvino domains for tracing
+ * @file itt.hpp
+ */
+
+#pragma once
+
+#include <openvino/cc/selective_build.h>
+#include <openvino/itt.hpp>
+
+namespace ngraph {
+namespace pass {
+namespace itt {
+namespace domains {
+    OV_ITT_DOMAIN(IETransform);
+}   // namespace domains
+}   // namespace itt
+}   // namespace pass
+}   // namespace ngraph
+
+OV_CC_DOMAINS(ngraph_pass);
+OV_CC_DOMAINS(ie_op);
+
+/*
+ * MATCHER_SCOPE macro allows to define and disable the section of code which we want to switch off if it is not used
+ * TRANSFORMATION_SCOPE macro allows to disable the code sections which are not needed if we disable the matcher
+ * INTERNAL_OP_SCOPE macro allows to disable parts of internal nGraph operations if they are not used
+ */
+#if defined(SELECTIVE_BUILD_ANALYZER)
+#define MATCHER_SCOPE(region) OV_SCOPE(ngraph_pass, region)
```

Comment:
From naming perspective `TRANSFORMATION_SCOPE` seems to be macro with logging, `MATCHER_SCOPE` macro based on callback log (in one place, see my previous comment). Vice versa.

Reply:
I changed the name of this macro